main.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-//go:generate sh generate_examples.sh
+//go:generate sh -c "cd _examples && go generate ./..."
```

Comment:
I originally had something like this, but I vaguely recall it caused a bunch of problems. I can't remember the details (something about relative paths? windows? aaaarg) and I've not made the time to dig it up, but sorry for the delay. 

This may work fine as is, and my old version was subtly wrong. I'll try to make some time on a windows machine to play around.